FirstProject/assets/new-evaluation.less

```diff
@@ -154,14 +170,175 @@
   width: 1200px;
   margin: 0 auto;
   margin-top: 50px;
+}
 
-  .col-md-3 {
-    display: inline-block;
-  }
+.col-md-3 {
+  display: inline-block;
 }
 
 a.border_bottom:hover {
-  border-bottom: 4px solid blue;
+  border-bottom: 4px solid @blue;
   border-bottom-width: 7px;
 }
 
+
+
+
+
+.language_form {
+  margin-top: 70px;
+}
+
+.description_section {
+  margin-left: 300px;
+}
+
+fieldset.field_section {
+  text-align: center;
+}
+
+legend {
+  text-align: left;
+  color: #03A9F4;
+}
+
+.input-circle[type='radio']{
+  width: 35px;
+  height: 23px;
+  border-radius: 15px;
+  top: -2px;
+  left: -1px;
+  position: relative;
+  background-color: #d1d3d1;
+  content: '';
+  display: inline-block;
+  visibility: visible;
+  border: 2px solid white;
+}
+
+.footer_area {
+  padding: 25px 350px 20px 0;
+  font-weight:bold;
+  font-size:16px;
+  color: gray;
+  text-align: right;
+  margin-top: 120px;
+  margin-bottom: 0;
+}
+
+
+
+
+.language_form {
+  margin-top: 70px;
+}
+
+.description_section {
+  margin-left: 300px;
+}
+
+fieldset.field_section {
+  text-align: center;
+}
+
+legend {
+  text-align: left;
+  color: #03A9F4;
+}
+
+.input-circle[type='radio']{
+  width: 35px;
+  height: 23px;
+  border-radius: 15px;
+  top: -2px;
+  left: -1px;
+  position: relative;
+  background-color: #d1d3d1;
+  content: '';
+  display: inline-block;
+  visibility: visible;
+  border: 2px solid white;
+}
+
+.footer_area {
+  padding: 25px 350px 20px 0;
+  font-weight:bold;
+  font-size:16px;
+  color: gray;
+  text-align: right;
+  margin-top: 120px;
+  margin-bottom: 0;
+}
+
+.logout_button {
+  float:right;
+}
+
+fieldset ul li {
+  margin: 0 auto 10px;
+}
+
+ul li label {
+  padding: 25px;
+  font-size: 18px;
+}
+
+fieldset legend h2 {
+  padding-right: 17px;
+  padding-left: 17px;
+}
+
+.col-md-3.bullets_menu {
+  margin: 0 auto 10px 10px;
+}
+
+ul li select {
+  margin-top: 10px;
+}
+
+/*Header form*/
+header form {
+  margin-left: 400px;
+}
+
+
+
+
+.logout_button {
+  float:right;
+}
+
+fieldset ul li {
```

Comment:
Is not required. I could add a class for the ul.
In this way I can make the CSS code reusable.